src/main/java/Tarnia.java

```diff
@@ -0,0 +1,64 @@
+import java.util.Scanner;
+
+public class Tarnia {
+    public static void main(String[] args) {
+        System.out.println("Hello! I'm 𝓣𝓪𝓻𝓷𝓲𝓪 💔");
+        System.out.println("What can I do for you?");
+
+        Scanner in = new Scanner(System.in);
+        Task[] tasks = new Task[100];
+        int counter = 0;
+
+        while (true) {
+            String line = in.nextLine().trim();
+            // split into command and index
+            String[] parts = line.split(" ", 2);
+            String command = parts[0];
+
+            // Switch is more efficient for these tasks, changed from if/else statements
+            switch (command) {
+            case "list":
+                System.out.println("Here are the tasks in your list! :");
+                for (int i = 0; i < counter; i++) {
+                    System.out.println((i + 1) + ". "
+                            + "[" + tasks[i].getStatusIcon() + "] "
+                            + tasks[i].getDescription());
+                }
+                break;
+
+            case "mark":
+                int mark_index = Integer.parseInt(parts[1]) - 1;
```

Comment:
mark_index, unmark_index use snake_case; Java conventions prefer markIndex, unmarkIndex.

```suggestion
                int markIndex = Integer.parseInt(parts[1]) - 1;
```